app/services/analytics_events.rb

```diff
@@ -2011,6 +2011,19 @@ def idv_doc_auth_verify_submitted(
     )
   end
 
+  # The JSON body of the response returned from Threatmetrix. PII has been removed.
+  # @param [Hash] response_body The response body returned by ThreatMetrix
+  def idv_doc_auth_verify_threatmetrix_response_body(
```

Comment:
I kinda think this should be named `idv_threatmetrix_response_body`? It's not doc auth specific, and that word "verify" is just hanging out in there

Reply:
Gotcha, yeah I was trying to match the pattern we use for the other events we fire here, but you are right that doc auth is not part of the equation so we can yank it. Willfix